recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/sndbe/sndbe_git.bbappend

```diff
@@ -5,4 +5,4 @@ SRCREV_rcar = "${AUTOREV}"
 
 SRC_URI_append_rcar = " git://github.com/xen-troops/snd_be.git;protocol=https;branch=master"
 
-EXTRA_OECMAKE_append_rcar = " -DWITH_DOC=OFF -DWITH_PULSE=ON"
+EXTRA_OECMAKE_append_rcar = " -DWITH_DOC=OFF -DWITH_PULSE=OFF -DWITH_ALSA=ON"
```

Comment:
I see no reasoning for switching from PA to ALSA. Neither commit message says anything about it

Reply:
I do not see vsnd functional either.
That is the setup @arminn does work now.

Reply:
No one has this knowledge though. My point was it needs to be documented

Reply:
OK, let us do it in a different way. I drop vsnd patches. It's on you to provide DOD capable functional  vsnd.